tools/nl/embeddings/input/base/sheets_svs.csv

```diff
@@ -3541,7 +3541,7 @@ WHO/mslv_Rural,Measles Immunization Coverage Among 1 year old in Rural Areas
 WHO/mslv_Urban,Measles Immunization Coverage Among 1 year old in Urban areas
 WHO/NCD_BMI_18A,Prevalence Of Underweight Among Adults
 WHO/NCD_BMI_30A,Prevalence Of Obesity Among Adults
-WHO/NCD_PAC,Prevalence Of Insufficient Physical Activity Among Adults

```

Comment:
Not sure insufficient physical activity is same as physical inactivity?

Based on https://screenshot.googleplex.com/AM63bQgSndLPMjf

Reply:
reverted this